http-gateway/web/src/components/table/utils.js

```diff
@@ -0,0 +1,12 @@
+// Case insensitive sort function
+export const compareIgnoreCase = (a, b) => {
+  let r1 = a.toLowerCase()

```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare - looks like this is a native function that can take care of the weird things about languages.

Reply:
I will look at it in the future, it needs bigger research since the browser support is not the best. Also, sorting will be asynchronous soon, so there is no need for us to spend that much time on it in JS.